pom.xml

```diff
@@ -0,0 +1,703 @@
+            <dependency>
+              <groupId>com.github.TBD54566975</groupId>
+              <artifactId>tbd-detekt-rules</artifactId>
+              <version>0.0.2</version>
+            </dependency>
+          </dependencies>
+        </plugin>
+        <plugin>
+          <groupId>io.github.martinvisser</groupId>
+          <artifactId>kover-maven-plugin</artifactId>
+          <version>${version.io.github.martinvisser}</version>
+          <configuration>
+            <!-- Rules are optional, but if none are configured the plugin cannot verify the coverage -->
+            <!-- So we set them to something very low as to not fail builds -->
+            <rules>
+              <rule>
+                <minValue>10</minValue>
+                <metric>LINE</metric>
+                <aggregation>COVERED_PERCENTAGE</aggregation>
+              </rule>
+              <rule>
+                <minValue>10</minValue>
+                <metric>BRANCH</metric>
+                <aggregation>COVERED_PERCENTAGE</aggregation>
+              </rule>
+            </rules>
+          </configuration>
+        </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>exec-maven-plugin</artifactId>
+          <version>3.2.0</version>
+        </plugin>
+        <plugin>
+          <groupId>org.jetbrains.kotlin</groupId>
+          <artifactId>kotlin-maven-plugin</artifactId>
+          <version>${version.kotlin}</version>
+          <configuration>
+            <jvmTarget>${kotlin.jvm.target}</jvmTarget>
+          </configuration>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-release-plugin</artifactId>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-source-plugin</artifactId>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-deploy-plugin</artifactId>
+      </plugin>
+      <plugin>
+        <groupId>io.github.martinvisser</groupId>
+        <artifactId>kover-maven-plugin</artifactId>
+      </plugin>
+      <plugin>
+        <groupId>com.github.ozsie</groupId>
+        <artifactId>detekt-maven-plugin</artifactId>
+      </plugin>
+      <!--
+      This is needed because the maven-release-plugin does not check out
+      submodules, needed for Web5 Test Vectors
+
+      Idea from: https://sumit-jha.medium.com/how-to-make-maven-release-plugin-work-with-git-submodules-917825a253f2
+      -->
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <inherited>false</inherited>
+        <executions>
+          <execution>
+            <id>clone git submodule</id>
+            <phase>initialize</phase>
+            <configuration>
+              <executable>git</executable>
+              <arguments>
```

Comment:
git commands but in xml 😂

Reply:
how was this done in gradle?

Reply:
Seems that gradle didnt need it, its a `maven-release-plugin` thing according to the comments above

Reply:
Maven plugin for Release does not handle submodules so we need to hack as documented above this section to get the submodules checked out before a release build.